lib/parser/file_ref.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Thrift.Parser.FileRef do
+  @type thrift_include :: String.t
+  @type t :: %__MODULE__{path: String.t, include_name: String.t, contents: String.t}
+  defstruct path: nil, include_name: nil, contents: nil
+
+  def new(path) do
+    thrift_file = """
```

Comment:
I'm open to suggestions. IIRC this was your idea. 

put it at the end of the file?